.github/workflows/ci-components-nightly.yml

```diff
@@ -25,15 +26,26 @@ jobs:
           node-version: ${{ env.NODE_VERSION }}
           cache: yarn
           cache-dependency-path: yarn.lock
+
       - name: Install Dependencies
         run: yarn install --immutable
+        
+      - name: Set up date variable
+        run: echo "DATE=$(date +'%Y-%m-%d')" >> $GITHUB_ENV
+
+      - name: Create directory for test results if one does not already exist
+        working-directory: packages/components
+        run: mkdir -p test-output
+
       - name: Run Tests
         run: yarn run test:a11y
```

Comment:
I think we need another step here to write the results into the file (instead of printing them on the console)

Trivial example
```
run: yarn run test:a11y > packages/components/test-output/a11y-test-results-${{ env.DATE }}.txt

Reply:
Isn't that what line 45 would be doing?

Reply:
ok, I have added it (but note the path, since we've already indicated working directory I don't think we need it again)